github/actions_artifacts.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// Artifact reprents a GitHub artifact.  Artifacts allow sharing
+// data between jobs in a workflow and provide storage for data
+// once a workflow is complete.
+//
+// GitHub API docs: https://developer.github.com/v3/actions/artifacts/
+type Artifact struct {
+	ID                 *int64  `json:"id,omitempty"`
+	NodeID             *string `json:"node_id,omitempty"`
+	Name               *string `json:"name,omitempty"`
+	SizeInBytes        *int64  `json:"size_in_bytes,omitempty"`
+	ArchiveDownloadURL *string `json:"archive_download_url,omitempty"`
+	Expired            *bool   `json:"expired,omitempty"`
+	CreatedAt          *string `json:"created_at,omitempty"`
```

Comment:
This and the next field need to be of type `*Timestamp`, please.

Reply:
Fixed.